sample-apps/dotnet/dotnet-ec2-win-main-setup.ps1

```diff
@@ -0,0 +1,99 @@
+param (
+    [string]$GetCloudwatchAgentCommand,
+    [string]$GetAdotDistroCommand,
+    [string]$GetSampleAppCommand,
+    [string]$TestId,
+    [string]$RemoteServicePrivateEndpoint,
+    [string]$TestCanaryType,
```

Comment:
What is this used for? I see it's passed to the traffic generator but IIRC it's not required.

Reply:
it's here because canary_type is in python and java, should have no use for it but not sure why other leave it here

Reply:
But removed, seems no influence